### DISCUSSION TITLE
Node Client for AMQP 1.0

### DISCUSSION THREAD

**Discussion by gperdomor** [rabbitmq/rabbitmq-website#2110] [General]
### Is your feature request related to a problem? Please describe.

With the introduction of AMQP 1.0 as a native protocol in RabbitMQ 4.0 I think that most users wants to migrate to the new version... As a Node developer, the most common library is `amqplib` (and also the recommended one in RabbitMQ website)...

Also with v4.0, was released a new set of client libraries for Java and .NET

There is a plan to develop new official libraries for other languages like Node, Swift, PHP, etc?

### Describe the solution you'd like

Update https://www.rabbitmq.com/client-libraries/amqp-client-libraries to add recommended libraries for AMQP 1.0

**Comment by michaelklishin** [comment]
@gperdomor please never use issues for questions in repositories that have discussions enabled.

AMQP 1.0 is not a "new version", it is a completely different protocol from AMQP 0-9-1 and a different data model which was developed after RabbitMQ has reached 1.0 with AMQP 0-8-0 and later AMQP 0-9-1 being its first protocols. Both AMQP 1.0 and AMQP 0-9-1 have their pros and cons.

There are existing AMQP 1.0 clients for many languages Anyone is welcome to develop a new one for any language they like. There generally won't be much RabbitMQ-specific.

Out of 10+ languages covered in AMQP 0-9-1 tutorials, all clients besides three (Java, .NET, Erlang) are community projects.
